File: University/lib/Dto/Base/LookupHeaderDto.dart
class LookupHeaderDto{
  String HeaderName = "";
  String HeaderType = "";
  String HeaderFormat = "";
  String HeaderWidth = "";

  LookupHeaderDto({
    this.HeaderName = "",
    this.HeaderType = "",
    this.HeaderFormat = "",
    this.HeaderWidth = ""
  });

  factory LookupHeaderDto.fromJson(Map<String, dynamic> json) {
    return LookupHeaderDto(
      HeaderName: json["HeaderName"] ?? "",
      HeaderType: json["HeaderType"] ?? "",
      HeaderFormat: json["HeaderFormat"] ?? "",
      HeaderWidth: json["HeaderWidth"] ?? "",
    );
  }

  Map toMap() {
    var map = <String, dynamic>{};
    map["HeaderName"] = HeaderName;
    map["HeaderType"] = HeaderType;
    map["HeaderFormat"] = HeaderFormat;
    map["HeaderWidth"] = HeaderWidth;

    return map;
  }
}

File: University.Service/University.Dto/Base/LookupHeaderDto.cs
using System;
using System.Data;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace University.Dto.Base
{
    [DataContract]
    public class LookupHeaderDto
    {
        [DataMember]
        public string HeaderName { get; set; }
        [DataMember]
        public string HeaderType { get; set; }
        [DataMember]
        public string HeaderFormat { get; set; }
        [DataMember]
        public string HeaderWidth { get; set; }
    }

    public class LookupHeaderMappingDto : Mapper<LookupHeaderDto>
    {
        protected override LookupHeaderDto PopulateItem(IDataRecord dr)
        {
            LookupHeaderDto dto = new LookupHeaderDto();
            for (int i = 0; i < dr.FieldCount; i++)
            {
                MapProperty(dto, dr.GetName(i), dr[i]);
            }
            return dto;
        }
    }
}
